rtdata/languages/default

```diff
@@ -4331,6 +4383,11 @@ TP_WAVELET_OPACITYWL_TOOLTIP;Modify the final local contrast at the end of the w
 TP_WAVELET_PASTEL;Pastel chroma
 TP_WAVELET_PROC;Process
 TP_WAVELET_PROTAB;Protection
+TP_WAVELET_PYRWAVTRC;Contrast profile
+TP_WAVELET_PYRWAVTRC_TOOLTIP;The 'Contrast profile' slider lets you select a series of presets that will significantly modify the effect of the “Variable contrast” curve. The presets range from least to most aggressive in ascending order.\n\nFor the upper preset levels, the visibility of the result in the preview window will depend on the image size and the system configuration. Increase the preview window size if this is the case.
+TP_WAVELET_PYRWAVTRC_CURVE_TOOLTIP;The Variable contrast curve takes into account the information provided by the presets. The mid-point of the x-axis corresponds to the predominant contrast value, irrespective of the detail level and the selected preset. The maximum point, to the right of the x-axis, corresponds to the maximum contrast value. It is recommended to reduce the variable contrast beyond the mid-point.
+TP_WAVELET_PYRWAVTRC_RESID_TOOLTIP;Residual contrast, adjusts the residual image after decomposition to compensate for excessive effects linked to the action of the curve and the presets (relief effect, contrast differences between structure and uniform areas).
```

Comment:
I'm having a hard time understanding the second half of this tooltip, and I'm sure others will feel this way. For example, what does it mean for the effects to be excessive? Too much contrast? What is "relief effect"? How are the contrast difference between structure (detailed?) and uniform areas problematic? How does the residual contrast compensate for these?

If it was up to me, I would simply explain what the residual contrast does and let the user decide if it helps solve an issue they have. Something like this:
```suggestion
TP_WAVELET_PYRWAVTRC_RESID_TOOLTIP;Adjusts the contrast for all detail levels larger than those covered by the selected Contrast profile.
```

Reply:
Apart from my bad English, I prefer the first version.
Here again, how to write (quite) complex things simply.

The "relief effect" in my bad English translates the "structure", the result of the decomposition. If the user has forced too much on the decomposed part the image will give the impression of being a sculpture. The residual part allows (in part) to attenuate this effect